test/jasmine/tests/geo_interact_test.js

```diff
@@ -14,14 +14,24 @@ describe('Test geo interactions', function() {
 
     describe('mock geo_first.json', function() {
         var mock = require('@mocks/geo_first.json');
+        var gd;
+
+        function mouseEventScatterGeo(type) {
+            mouseEvent(type, 300, 235);
```

Comment:
It still surprises me that this isn't a more widely found solution when searching for "mouse event testing".